cmd/bosun/web/web.go

```diff
@@ -99,6 +99,7 @@ func Listen(listenAddr string, devMode bool, tsdbHost string) error {
 	router.Handle("/api/metadata/put", JSON(PutMetadata))
 	router.Handle("/api/metric", JSON(UniqueMetrics))
 	router.Handle("/api/metric/{tagk}/{tagv}", JSON(MetricsByTagPair))
+	router.Handle("/api/metric/keytree", JSON(TagKeysByMetricTree))
```

Comment:
Naming here is harder yet. `keytree` is not super descriptive to me. You can call it what you want though.